python/ray/services.py

```diff
@@ -540,17 +539,15 @@ def wait_for_redis_to_start(redis_ip_address,
         redis_ip_address (str): The IP address of the redis server.
         redis_port (int): The port of the redis server.
         password (str): The password of the redis server.
-        num_retries (int): The number of times to try connecting with redis.
-            The client will sleep for one second between attempts.
 
     Raises:
         Exception: An exception is raised if we could not connect with Redis.
     """
     redis_client = redis.StrictRedis(
         host=redis_ip_address, port=redis_port, password=password)
     # Wait for the Redis server to start.
-    counter = 0
-    while counter < num_retries:
+    num_retries = 5000
```

Comment:
Actually, is there a reason to not have `num_retries` as an arg anymore?

Reply:
My reasoning was that we may want to change the meaning of `num_retries`, e.g., in this PR we are changing it from seconds to milliseconds, and that could break things for callers who are using that argument.